packages/reporter/src/std/StdReporter.ts

```diff
@@ -0,0 +1,123 @@
+import { Reporter } from '../lib';
+import { Issue, Scan, Severity } from '../models';
+import { IssuesGrouper } from '../utils';
+import table, { Header } from 'tty-table';
+import chalk from 'chalk';
+
+export class StdReporter implements Reporter {
+  private readonly severityColorFn: Record<
+    Severity,
+    (...x: unknown[]) => string
+  > = {
+    [Severity.HIGH]: chalk.red,
+    [Severity.MEDIUM]: chalk.yellow,
+    [Severity.LOW]: chalk.blue
+  };
+
+  private readonly severityPrintFn: Record<
+    Severity,
+    (...x: unknown[]) => void
+  > = {
+    /* eslint-disable no-console */
+    [Severity.HIGH]: console.error,
+    [Severity.MEDIUM]: console.warn,
+    [Severity.LOW]: console.log
+    /* eslint-enable no-console */
+  };
+
+  public async report(scan: Scan): Promise<void> {
+    const issues: Issue[] = await scan.issues();
+    if (!issues.length) {
+      return;
+    }
+
+    [Severity.HIGH, Severity.MEDIUM, Severity.LOW].forEach(
+      (severity: Severity) => {
+        const message = this.formatFindingsMessage(issues, severity);
+        if (message) {
+          this.print(message, severity);
+        }
+      }
+    );
+
+    // eslint-disable-next-line no-console
+    console.log(this.renderDetailsTable(issues));
+  }
+
+  private formatFindingsMessage(
+    issues: Issue[],
+    severity: Severity
+  ): string | undefined {
+    const filtered = issues.filter(issue => issue.severity === severity);
+    if (!filtered.length) {
+      return undefined;
+    }
+
+    return this.colorize(
+      `Found ${filtered.length} ${severity} severity ${this.pluralize(
+        'issue',
+        filtered.length
+      )}.`,
+      severity
+    );
```

Comment:
Avoid multiple return statements if it is possible
```suggestion
    if (filtered.length) {
      return this.colorize(
        `Found ${filtered.length} ${severity} severity ${this.pluralize(
          'issue',
          filtered.length
        )}.`,
        severity
      );
    }
```